src/static/css/module/tab.less

```diff
@@ -99,7 +99,7 @@
         font-size: 1em;
       }
       .tab-label {
-        display: none;
+        //display: none;
```

Comment:
I think we want to change this to a media query with a max-width of 332px, not just make it visible. At 320px (standard iphoneish) it will not fit and will wrap to a new line.